matplotlibrc.template

```diff
@@ -1,116 +1,116 @@
-# See http://matplotlib.org/api/artist_api.html#module-matplotlib.lines for more
-# information on line properties.
-#lines.linewidth   : 1.5     # line width in points
-#lines.linestyle   : -       # solid line
-#lines.color       : C0      # has no affect on plot(); see axes.prop_cycle
-#lines.marker      : None    # the default marker
-#lines.markeredgewidth  : 1.0     # the line width around the marker symbol
-#lines.markersize  : 6            # markersize, in points
-#lines.dash_joinstyle : miter        # miter|round|bevel
-#lines.dash_capstyle : butt          # butt|round|projecting
-#lines.solid_joinstyle : miter       # miter|round|bevel
-#lines.solid_capstyle : projecting   # butt|round|projecting
-#lines.antialiased : True         # render lines in antialiased (no jaggies)
-
-# The three standard dash patterns.  These are scaled by the linewidth.
-#lines.dashed_pattern : 2.8, 1.2
-#lines.dashdot_pattern : 4.8, 1.2, 0.8, 1.2
-#lines.dotted_pattern : 1.1, 1.1
+#### LINES
+## See http://matplotlib.org/api/artist_api.html#module-matplotlib.lines for more
+## information on line properties.
+#lines.linewidth   : 1.5     ## line width in points
+#lines.linestyle   : -       ## solid line
+#lines.color       : C0      ## has no affect on plot(); see axes.prop_cycle
+#lines.marker      : None    ## the default marker
+#lines.markeredgewidth  : 1.0     ## the line width around the marker symbol
+#lines.markersize  : 6            ## markersize, in points
+#lines.dash_joinstyle : round        ## miter|round|bevel
+#lines.dash_capstyle : butt          ## butt|round|projecting
+#lines.solid_joinstyle : round       ## miter|round|bevel
+#lines.solid_capstyle : projecting   ## butt|round|projecting
+#lines.antialiased : True         ## render lines in antialiased (no jaggies)
+
+## The three standard dash patterns.  These are scaled by the linewidth.
+#lines.dashed_pattern : 3.7, 1.6
+#lines.dashdot_pattern : 6.4, 1.6, 1, 1.6
+#lines.dotted_pattern : 1, 1.65
 #lines.scale_dashes : True
 
-#markers.fillstyle: full # full|left|right|bottom|top|none
+#markers.fillstyle: full ## full|left|right|bottom|top|none
 
-### PATCHES
-# Patches are graphical objects that fill 2D space, like polygons or
-# circles.  See
-# http://matplotlib.org/api/artist_api.html#module-matplotlib.patches
-# information on patch properties
-#patch.linewidth        : 1        # edge width in points.
+#### PATCHES
+## Patches are graphical objects that fill 2D space, like polygons or
+## circles.  See
+## http://matplotlib.org/api/artist_api.html#module-matplotlib.patches
+## information on patch properties
+#patch.linewidth        : 1        ## edge width in points.
 #patch.facecolor        : C0
-#patch.edgecolor        : black   # if forced, or patch is not filled
-#patch.force_edgecolor  : False   # True to always use edgecolor
-#patch.antialiased      : True    # render patches in antialiased (no jaggies)
+#patch.edgecolor        : k       ## if forced, or patch is not filled
```

Comment:
Is there a reason to exchange the long color names if favor of the short ones? IMO the long names are easier to understand, especially for someone not familiar with the matplotlib color system.

Reply:
I changed this to exactly the value used in defaultParams, which is `"k"` or `"w"` instead of `"black"` or `"white"`, this makes it now easy to identify which params have non-matching values compared to the default as stated above. Should I change it back?

Reply:
I see the benefit of using the exact same values. From a user point of view, I would suggest to change the defaultParams to the long form. However I'm not an expert on matplotlib.rcParams and don't know if that has any negative side effects. This should be judged by someone with more experience in this field. I'll open an issue.

Reply:
Discussion here #10550. If this would get resolved immediately, I suggest to change the code of this PR back. Otherwise it's ok to start with the short forms in the template and a PR on #10550 would change them back.